kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OddNumbersExterminator {
+
+    public List<Integer> exterminate(List<Integer> numbers) {
+
+        List<Integer> resultNumbers = new ArrayList<>();
+        int exterminatedNumbers = 0;
+
+        for (Integer temp : numbers) {
+
+            if (temp % 2 == 0) {
+                resultNumbers.add(temp);
+            } else {
+                exterminatedNumbers++;
```

Comment:
exterminatedNumbers jest nigdzie nie wykorzystane. Mozesz albo:
- zwracac te informacje wraz z lista (w klasie ktora mialaby te 2 pola)
- printowac to
- wyrzucic te zmienna